ccd-definition/AuthorisationCaseField/CareSupervision/eps-managing.json

```diff
@@ -0,0 +1,1024 @@
+[
```

Comment:
matches 1 to 1 to la-solicitor.json apart from the generic fields assigned to caseworker-publiclaw-solicitor ✅